Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM rockylinux:9
-MAINTAINER APEL Administrators <apel-admins@stfc.ac.uk>
+LABEL org.opencontainers.image.authors="apel-admins@stfc.ac.uk" \
```

Comment:
According to [this](https://github.com/Shopify/docker/blob/master/docs/userguide/labels-custom-metadata.md#) you should try to not do that

> Docker recommends you add multiple labels in a single LABEL instruction. Using individual instructions for each label can result in an inefficient image. This is because each LABEL instruction in a Dockerfile produces a new IMAGE layer.